Dockerfile

```
@@ -61,19 +61,32 @@ RUN apt-get update -qq && \
     apt-get install --no-install-recommends -y curl libpq-dev libvips libjemalloc2 && \
     apt-get clean
 
+ENV USERNAME rails_api_base
```

Maybe this way we won't need to change if for all the projects 🙂 
```suggestion
ENV USERNAME rails
```